docs/content/latest/api/ysql/datatypes/type_datetime/date-time-data-types-semantics/type-interval/interval-arithmetic/moment-interval-overloads-of-plus-and-minus.md

```diff
@@ -257,7 +254,7 @@ Notice that the date for the third result, _9-April_, is different from that for
 You might be tempted to speculate about priority rules for how the fields of a hybrid _interval_ value are acted on. Don't do this. The PostgreSQL documentation doesn't state the rules and application code that used hybrid _interval_ values would be hard to understand and likely, therefore, to be unreliable.
 
 {{< tip title="Avoid arithmetic that uses hybrid interval values." >}}
-Yugabyte recommends that you avoid creating and using _hybrid interval_ values; rather, you should adopt the practice that the section [Defining and using custom domain types to specialize the native _interval_ functionality](../../custom-interval-domains/) explains.
+Yugabyte recommends that you avoid creating and using _hybrid interval_ values; rather, you should adopt the practice that the section [Custom domain types for specializing the native _interval_ functionality](../../custom-interval-domains/) explains.
```

Comment:
You switch between British/Canadian and American spelling... I do too. Watch for it, and at least be consistent within a given page. :)